ipaserver/install/krbinstance.py

```diff
@@ -451,6 +451,30 @@ def pkinit_enable(self):
         service.set_service_entry_config(
             'KDC', self.fqdn, [PKINIT_ENABLED], self.suffix)
 
+    def pkinit_disable(self):
+        """
+        unadvertise enabled PKINIT feature in master's KDC entry in LDAP
+        """
+        ldap = api.Backend.ldap2
+        dn = DN(('cn', 'KDC'),
+                ('cn', self.fqdn),
```

Comment:
Yo can use `DN(('cn', 'KDC'), api.Object.master.get_dn(self.fqdn))` to construct service DN in a cleaner way using existing abstractions.